README.md

```diff
@@ -0,0 +1,85 @@
+# Django Lightweight Queue
+
+DLQ is a lightweight & modular queue and cron system for Django.
+
+## Basic Usage
+
+``` python
+import time
+from django_lightweight_queue import task
+
+# Define a task
+@task()
+def long_running_task(first_arg, second_arg):
+    time.sleep(first_arg * second_arg)
+
+# Request that the task be executed at some point
+long_running_task(4, second_arg=9)
+```
+
+See the docstring on the [`task`](django_lightweight_queue/task.py) decorator
+for more details.
+
+## Configuration
+
+Configuration options should be placed in the usual Django settings files,
+prefixed with `LIGHTWEIGHT_QUEUE_`, for example:
```

Comment:
The problem is that you've conflated "placed in the usual Django settings files" (which, if you disobey, it completely won't work) and "prefixed with `LIGHTWEIGHT_QUEUE_`" (which, if you disobey, has zero impact). It therefore implies (or invites the reader to infer) that the choice of name is significant.